executor/modes/local/websocket_client.py

```
@@ -0,0 +1,530 @@
+        system = platform.system()
+
+        try:
+            if system == "Darwin":
+                # macOS: Use IOPlatformUUID (Hardware UUID)
+                result = subprocess.run(
+                    [
+                        "ioreg",
+                        "-rd1",
+                        "-c",
+                        "IOPlatformExpertDevice",
+                    ],
```

Ruff (0.14.14)

Starting a process with a partial executable path

(S607)